go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/stackrox/rox
 
-go 1.19
+go 1.20
```

Comment:
why were all of these bumped? I thought there was some kind of issue with downstream when we did this?

Reply:
I've tried everything to make it working with nix, so my last hope was that maybe it's related to go version but it's not. I'll close the PR as I'm not a mac user and it's too hard for me to test it.